ansible/roles_ocp_workloads/ocp4_workload_quarkus_super_heroes_demo/tasks/deploy_cicd.yaml

```diff
@@ -0,0 +1,270 @@
+  kubernetes.core.k8s:
+    state: present
+    definition: "{{ lookup('template', 'tekton/{{ item }}.yaml.j2') }}"
+  loop:
+  - pvc
+  - rolebinding
+  - task-s2i-binary
+  - task-update-kustomize
+  - task-generate-tag
+
+- name: "[{{ t_project_name_pipelines }}] - create tekton pipeline quarkus-superheroes"
+  vars:
+    _app_name: rest-fights
+    # yamllint disable-line rule:line-length
+    _source_git_url: "{{ r_gitea.resources[0].status.giteaRoute }}/{{ ocp4_workload_quarkus_super_heroes_demo_clone_repo_gitea_user }}/{{ ocp4_workload_quarkus_super_heroes_demo_clone_repo_gitea_repo_name }}"
+    _source_git_revision: "{{ ocp4_workload_quarkus_super_heroes_demo_release_tag }}"
+  kubernetes.core.k8s:
+    state: present
+    definition: "{{ lookup('template', 'tekton/pipeline.yaml.j2') }}"
+
+- name: "[{{ t_project_name_pipelines }}] - create tekton triggers"
+  kubernetes.core.k8s:
+    state: present
+    definition: "{{ lookup('template', 'tekton/{{ item }}.yaml.j2') }}"
+  loop:
+  - trigger-template
+  - trigger-binding
+  - trigger-eventlistener
+  - trigger-eventlistener-route
+
+- name: create gitea webook
+  ansible.builtin.uri:
+    # yamllint disable-line rule:line-length
+    url: "{{ r_gitea.resources[0].status.giteaRoute }}/api/v1/repos/{{ ocp4_workload_quarkus_super_heroes_demo_clone_repo_gitea_user }}/{{ ocp4_workload_quarkus_super_heroes_demo_clone_repo_gitea_repo_name }}/hooks"
+    method: POST
+    force_basic_auth: true
+    user: "{{ ocp4_workload_quarkus_super_heroes_demo_clone_repo_gitea_user }}"
+    password: "{{ ocp4_workload_quarkus_super_heroes_demo_clone_repo_gitea_password }}"
+    headers:
+      accept: application/json
+    body_format: json
+    body: |
+      {
+        "active": true,
+        "branch_filter": "*",
+        "config": {
+          "url": "http://el-{{ t_project_name_pipelines }}.{{ route_subdomain }}",
+          "http_method": "POST",
+          "content_type": "json"
+        },
+        "events": [
+          "push"
+        ],
+        "type": "gitea"
+      }
+    status_code: [201]
+
+- name: "[{{ t_project_name_pipelines }}] - delete pipelinerun quarkus-superheroes"
+  kubernetes.core.k8s:
+    state: absent
+    api_version: tekton.dev/v1beta1
+    kind: PipelineRun
+    name: quarkus-superheroes
+    namespace: "{{ t_project_name_pipelines }}"
+
+- name: "[{{ t_project_name_pipelines }}] - create pipelinerun quarkus-superheroes"
+  kubernetes.core.k8s:
+    state: present
+    api_version: tekton.dev/v1beta1
+    kind: PipelineRun
+    definition:
+      metadata:
+        name: quarkus-superheroes
+        namespace: "{{ t_project_name_pipelines }}"
+      spec:
+        pipelineRef:
+          name: quarkus-superheroes
+        workspaces:
+        - name: app-source
+          persistentVolumeClaim:
+            claimName: workspace-pvc
+        - name: maven-settings
+          emptyDir: {}
+
+- name: "[{{ t_project_name_pipelines }}] - wait pipelinerun quarkus-superheroes"
```

Comment:
wait for ?

Reply:
Will add the "for" preposition in the task name.